airbyte-integrations/connectors/source-file-secure/Dockerfile

```diff
@@ -1,3 +1,11 @@
+### WARNING ###
+# This Dockerfile will soon be deprecated.
+# It is not used to build the connector image we publish to DockerHub.
+# The new logic to build the connector image is declared with Dagger here:
+# https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py#L771
+
+# If you need to add a custom logic to build your connector image, you can do it by adding a finalize_build.sh or finalize_build.py script in the connector folder.
+# Please reach out to the Connectors Operations team if you have any question.
```

Comment:
`source-file-secure` is now fully built from Dagger.  As `source-file` needs to be mounted at build time it was easier to accomplish it with Dagger.